### PR TITLE
Vetted System conversion to SQL

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

This PR will convert the vetted system to SQL. It will populate the with entries from the old config file, and then that file can be safely removed after it is imported. Also adds three verbs, one to force a conversion, one to add, and one to remove a ckey from the database. The admin who adds a user is record in the DB. 

## Proof Of Testing
heidisql_086ihiIsfK

## Changelog

:cl:
refactor: Vetted system is now using SQL and improved. 
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I'm finally free

File: modular_zubbers/code/modules/vetted/vetted.dm [hunk 1]
```diff
@@ -37,20 +38,66 @@ GLOBAL_PROTECT(vetted_list)
...
+		SSplayer_ranks.vetted_controller.convert_all_to_sql()
+
+ADMIN_VERB(add_vetted, R_ADMIN, "Add user to Vetted", "Adds a user to the vetted list", ADMIN_CATEGORY_MAIN)
+	var/user_adding = tgui_input_text(usr, "Whom is being added?", "Vetted List")
+	if(length(user_adding))
+		SSplayer_ranks.vetted_controller.add_player(ckey = user_adding, admin = usr)
+ADMIN_VERB(remove_vetted, R_ADMIN, "Remove user from Vetted", "Removes a user from the vetted list", ADMIN_CATEGORY_MAIN)
+	var/user_del = tgui_input_text(usr, "Whom is being Removed?", "Vetted List")
+	if(length(user_del))
+		SSplayer_ranks.vetted_controller.remove_player(ckey = user_del)
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/vetted/vetted.dm:103]: make sure these procs give admin_log feedback.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

**Comment by Cyprex** [comment]
Should increment schema version

**Comment by StrangeWeirdKitten** [comment]
Merging due to the importance of it.

**Pull request merged**